lighthouse-cli/run.js

```diff
@@ -123,6 +123,9 @@ function saveResults(results, artifacts, flags) {
 
   if (flags.saveAssets) {
     promise = promise.then(_ => assetSaver.saveAssets(artifacts, results.audits, resolvedPath));
+  } else if (flags.gatherMode || flags.auditMode) {
+    const latestPath = path.join(cwd, 'latest-run', 'lhr.json');
+    promise = promise.then(_ => Printer.write(results, Printer.OutputMode.json, latestPath));
```

Comment:
will we be able to do this for just `gatherMode`? seems like we can only do this with auditMode or `gatherMode === auditMode`

Reply:
actually yah this should instead be 

 ```js
else if (flags.auditMode) {
```